src/utils/IO.js

```diff
@@ -0,0 +1,17 @@
+const MissionUtils = require("@woowacourse/mission-utils");
```

Comment:
```suggestion
const { Console }= require("@woowacourse/mission-utils");
```
MissionUtils의 Console만 이용하니까 이렇게 써주고 밑에도 MissionUtils를 생략하면 더 좋을 것 같습니다!

Reply:
다른 곳에서도 구조분해할당으로 가져오는데 이부분에서도 구조분해할당으로 가져왔다면 일관성도 있고 더 좋을 것 같네요!